### PR TITLE
Added Effect Stop and stare

### PR DESCRIPTION
**Pull request by ProfessorBiddle**
Makes everyone stop in place/stop their vehicles and get out, and stare at the player.

### CODE REVIEW

**Comment by Last0xygen** [comment]
Isn’t this the same as „You are famous“?

**Comment by TurnrDev** [comment]
Not really, in You are Famous, they run towards you

**Comment by ProfessorBiddle** [comment]
Not really, in you are famous they continually run towards you and even
pull you out of vehicles, with this they just stop. And state. And crash
their helicopters.

File: ChaosMod/Effects/db/Peds/PedsStopAndStare.cpp [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#include <stdafx.h>
+static void OnStart()
```
Comment (pongo1231) [inline ChaosMod/Effects/db/Peds/PedsStopAndStare.cpp:2]: Don't forget to add your credits

**Pull request merged**